api/src/controllers/animeController.ts

```diff
@@ -0,0 +1,82 @@
+import type { Request, Response, NextFunction } from '@@types/express.js';
+import { errors, entities } from '@@utils/index.js';
+import { Anime } from '@@entities/index.js';
+
+interface Params {
+    animeId: number
+};
+
+// Create
+async function create(req: Request, res: Response<Params>, next: NextFunction) {
+    const { params } = res.locals;
+    const { animeId } = params as Params;
```

Comment:
You can't need to pass `as Params` here since the `Params` type is being passed into the `Response` type. In your text editor, hovering over `params` on line 11 should display the type `Params`